app/assets/stylesheets/semantic_style.scss

```diff
@@ -515,9 +523,9 @@ article.alert {
 }
 
 /*
- Current status: hints are hidden, but have not got the incantations
- to make them show.
- 8/20/2015
+Current status: hints are hidden, but have not got the incantations
```

Comment:
I find myself preferring the `# TODO: message here` pattern because it's super-easy to find in search/ack/grep